src/checkers/htmlchecker.py

```diff
@@ -109,6 +108,16 @@ def check(self, external_data):
 
         self._update_version(external_data, latest_version, abs_url)
 
+    def _substitute_placeholders(self, template, version):
```

Comment:
Remove `self` arg here and add `@staticmethod` decorator, since this method doesn't access `self`.